### DISCUSSION TITLE
Import Error in Main.py

### DISCUSSION THREAD

**Discussion by tonyhart7** [fastapi/fastapi#6470] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from sqlalchemy.orm import Session

from . import crud, models, schemas
from .database import SessionLocal, engine
```

### Description

this is my main.py file 

and as you can see I want to import crud,model and schemas by following official doc at https://fastapi.tiangolo.com/tutorial/sql-databases/

but I cant import all that why that happen ??

### Operating System

Windows

### Operating System Details

Windows 11

### FastAPI Version

0.70.1

### Python Version

Python 3.10.0

**Comment by tonyhart7** [comment]
more code error 
~~~ 
from . import crud, models, schemas
ImportError: attempted relative import with no known parent package
~~~

**Comment by ifamirhasan** [comment]
What is your files structure?
try this:

`from db import crud, models, schemas`
`from db.database import SessionLocal, engine`

**Comment by tonyhart7** [comment]
> What is your files structure? try this:
> 
> `from db import crud, models, schemas from db.database import SessionLocal, engine`

Iam following https://fastapi.tiangolo.com/tutorial/sql-databases/ like in this doc 

so my code structure is 
-app
---main.py
---models.py
---database.py
     etc

**Comment by ifamirhasan** [comment]
try this:
```
import crud, models, schemas
from database import SessionLocal, engine
```

**Comment by tonyhart7** [comment]
> try this:
> 
> ```
> import crud, models, schemas
> from database import SessionLocal, engine
> ```

this works turn out I must remove all dot ( . ) in all files like models, database and schemas
